test/integration/simulacron.js

```diff
@@ -0,0 +1,310 @@
+'use strict';
+var helper = require('../test-helper');
+var http = require('http');
+var spawn = require('child_process').spawn;
+var util = require('util');
+var fs = require('fs');
+var utils = require('../../lib/utils.js');
+
+var simulacronHelper = {
+  _execute: function(processName, params, cb) {
+    var originalProcessName = processName;
+
+    // If process hasn't completed in 10 seconds.
+    var timeout = undefined;
+    if(cb) {
+      timeout = setTimeout(function() {
+        cb('Timed out while waiting for ' + processName + ' to complete.');
+      }, 10000);
+    }
+
+    var p = spawn(processName, params, {});
+    p.stdout.setEncoding('utf8');
+    p.stderr.setEncoding('utf8');
+    p.stdout.on('data', function (data) {
+      helper.trace('%s_out> %s', originalProcessName, data);
+    });
+
+    p.stderr.on('data', function (data) {
+      helper.trace('%s_err> %s', originalProcessName, data);
+    });
+
+    p.on('close', function (code) {
+      helper.trace('%s exited with code %d', originalProcessName, code);
+      if(cb) {
+        clearTimeout(timeout);
+        if (code === 0) {
+          cb();
+        } else {
+          cb(Error('Process exited with non-zero exit code: ' + code));
+        }
+      }
+    });
+
+    return p;
+  },
+  start: function(cb) {
+    var self = this;
+    var simulacronJarPath = process.env['SIMULACRON_PATH'];
+    if (!simulacronJarPath) {
+      simulacronJarPath = '$HOME/simulacron.jar';
```

Comment:
This doesn't work it seems.  Need to resolve the `$HOME` environment variable from process.env otherwise it interprets `$HOME` literally

Reply:
went ahead and pushed a commit for that.